Urban Market/ViewModels/HomeScreenViewModel.swift

```diff
@@ -9,7 +9,8 @@ import Foundation
 import Combine
 
 class HomeScreenViewModel: ObservableObject {
-    
+    //testCommentTony
+    var apiString: String?
```

Comment:
Why have you done this?

Reply:
I needed the string to display on the screen

Reply:
Sorry you need to refactor @TonytLieu